NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty
# Subscription Feature and Button #30

**Description**
When you click on MyPosts, you can click the button on the top right to "Create New Post". You can fill out the title, subject, category, and hit "Save" and the post should update to your list of my posts. Similarly, a delete and edit button should be next to your posts. When you hit delete it will prompt you to delete the post and if you hit "Yes" it will be removed from your post list. You can hit the edit button and change one of the input fields, and hit save, it will change in your list of posts too.
[Trello Ticket #30](https://trello.com/c/jWUfn9n7)


**Testing Instructions**
- git add and git commit whatever you're working on on your branch
- git checkout main
- git fetch --a git checkout: at-subscribe-#30
- If you don't see the most recent version of my files, git pull origin: at-subscribe-#30
- If you haven't created the Tabloid MVC database, please do so.
- Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty/blob/main/SQL/01_Tabloid_Create_DB.sql)
- Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty/blob/main/SQL/02_Tabloid_Seed_Data.sql)
- Click the execute with debugger solid green button to launch the app
- In browser, please go to Login on the nav bar and log in with this email: [foo@bar.com](mailto:foo@bar.com)
- This should redirect you back to the home page. You should see a "Posts" option on the navbar (otherwise referred to as, 'Post Details')
- Click on the page and select an article, in which you would like to see its details
- On the page it directs you to, with the details of the post, you should see a button on the bottom that says "Subscribe to [author]"
- Click on that button once, and you are subscribed!
- I have not made a pop-up alert yet, but if you open your React Dev Tools and go to Components, PostDetails, and look at the state you will see it saved

**Checklist:**
